### ISSUE TITLE
Accessory information using MQTT topics

### ISSUE THREAD

**Issue by Denny5667** [arachnetech/homebridge-mqttthing#348]
Hi Guys, 

i have a feature request. Could you please implement a feature to tell HomeKit an accessory information, like "manufacturer", "serialNumber", "model",  "firmwareRevision" using MQTT topics? It would be very helpful to update this information dynamicaly, for example when the device firmware revision is changed.

Thanks in advance!

**Comment by samwelnella** [comment]
I agree it'd be great to get this information using MQTT topics if possible.

**Comment by NorthernMan54** [comment]
@Denny5667 @samwelnella homebridge-tasmota does this automagically using home assistant Mqtt discovery messages to create the device in homebridge
